tests/typecheck/test_annotated.yml

```diff
@@ -11,3 +12,20 @@
         @dataclass(unsafe_hash=True)
         class RatingComposite:
             max_value: Annotated[int, IntegerType(min_value=1, max_value=10)] = 5
+
+        class Obj(TypedDict):
+            foo: int
+
+        class X:
+            x: Annotated[Blog, Obj]
+        reveal_type(X().x)  # N: Revealed type is "myapp.models.Blog"
```

Comment:
```suggestion
        reveal_type(X().x)  # N: Revealed type is "myapp.models.Blog"
        reveal_type(X().x.foo)  # N: Revealed type is "builtins.int"
```

Reply:
A bit confusing here, but `.foo` shouldn't be recognised as an attribute. I'm just testing that our hook that listens to `Annotated` doesn't act on any kind of `Annotated`.

But I'll add in what the expected output is

Reply:
Oh sorry, you are right. `.foo` should not be an attribute.